app/main/forms.py

```diff
@@ -74,13 +80,73 @@ class MoveUserForm(FlaskForm):
         ])
 
 
+class NotSuffixOfBlacklist:
+    "WTForms validator, asserting that supplied value is not a prefix of any of the values present in a blacklist"
+
+    # path, relative to flask app root_path, to look for suffix blacklist files. all files found here will be read,
+    # one suffix per line
+    BLACKLIST_DIR_PATH: str = "data/public_domain_suffix_blacklist"
+
+    @staticmethod
+    def _normalized_suffix(suffix: str) -> str:
+        return (
+            # all suffixes we deal with implicitly begin with a separator - make that explicit here
```

Comment:
Can these inline comments just be a docstring?

> Return reversed suffix and ensure last character is a '.'

Also `        return (("" if suffix.startswith(".") else ".") + suffix.strip().lower())[::-1]` fits within line limit

Reply:
docstrings are for a different thing - to document the general thing the function does from an outside caller's point of view

Reply:
But yes, the shorter explanation will probably do.